frontend/package.json

```diff
@@ -36,45 +36,39 @@
     "clean-webpack-plugin": "4.0.0-alpha.0",
     "css-loader": "5.2.4",
     "d3": "3.5.17",
-
     "datatables.net": "1.10.13",
     "datatables.net-bs": "1.10.13",
-
     "enzyme": "3.11.0",
     "es6-promise": "4.2.8",
     "eslint": "7.25.0",
     "eslint-loader": "4.0.2",
     "eslint-plugin-react": "7.23.2",
     "expose-loader": "2.0.0",
-    "mini-css-extract-plugin": "1.6.0",
     "file-loader": "6.2.0",
     "glob": "7.1.6",
     "immutability-helper": "3.1.1",
     "imports-loader": "2.0.0",
     "jquery": "3.3.1",
     "jsdom": "11.1.0",
+    "mini-css-extract-plugin": "1.6.0",
     "mocha": "3.4.2",
     "moment": "2.19.1",
     "node-sass": "6.0.0",
     "nvd3": "1.8.6",
     "optimize-css-assets-webpack-plugin": "5.0.4",
-    "popper.js":"1.14.3",
+    "popper.js": "1.14.3",
     "prop-types": "15.7.2",
-
     "react": "16.14.0",
+    "react-bootstrap": "1.5.2",
     "react-dom": "16.14.0",
-    "react-test-renderer": "16.14.0",
-
+    "react-dropdown-select": "^4.7.4",
```

Comment:
So I'm not totally against the use of `^` here, but I'm also not quite sure if we should use `^`, `~`, or nothing. For differences between the two see: https://stackoverflow.com/questions/22343224/whats-the-difference-between-tilde-and-caret-in-package-json

Right now, all our other packages just specify the version, so for consistency we might want to keep it specific. On the other hand, it might be useful to get patches. @peridoteagle what do you think?

Reply:
After some discussion I think this is fine. We might want to make a branch to update all of the dependencies to use ^, but that's outside the scope of this PR